subsys/metrics/src/main/java/org/commonjava/indy/metrics/zabbix/reporter/IndyZabbixReporter.java

```diff
@@ -302,7 +301,7 @@ else if ( logger.isDebugEnabled() )
                 logger.info( "report metrics to zabbix success. " + senderResult );
             }
         }
-        catch ( IOException e )
+        catch ( Exception e )
```

Comment:
This is a very wide catch clause. Can we narrow it to specific checked exceptions and/or runtime exceptions we expect to encounter?